src/main/java/seedu/address/model/InternalState.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model;
+
+import seedu.address.model.person.Person;
+
+/**
+* This class holds the actual state of the application.*/
+public final class InternalState {
+    private int primaryKeyCounter;
```

Comment:
Can we change the variable name to be more specific to `Person` class? We eventually have to add a `primaryKeyCounter` for `Activity` class as well.